### PR TITLE
delete all instrument fields

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds an option to delete instrument fields.

### CODE REVIEW

**Review by guynir42 — COMMENTED** [review]
It looks like you are making the delete instrument do something it is really not supposed to be doing. Since deleting an instrument is a dangerous thing, I would think using the delete instrument handler for other things is not a good idea. I don't know if deleting the associated fields is something we might be doing often (e.g., when playing around with skymaps) so maybe it should go in its own handler?

File: skyportal/handlers/api/instrument.py [hunk 1]
```diff
@@ -975,3 +965,48 @@ def add_tiles(
...
         Session.remove()
         return field_ids
+
+
+class InstrumentFieldHandler(BaseHandler):
+    @permissions(['Delete instrument'])
+    def delete(self, instrument_id):
+        """
+        ---
+        description: Delete an instrument
```
Comment (guynir42) [inline skyportal/handlers/api/instrument.py:975]:
```suggestion
        description: Delete fields associated with an instrument
```

File: skyportal/handlers/api/instrument.py [hunk 2]
```diff
@@ -975,3 +965,48 @@ def add_tiles(
     finally:
         Session.remove()
         return field_ids
+
+
+class InstrumentFieldHandler(BaseHandler):
+    @permissions(['Delete instrument'])
```
Comment (guynir42) [inline skyportal/handlers/api/instrument.py:971]: Is this the correct permissions for this action?

**Review by guynir42 — APPROVED** [review]
Minor comments:

File: skyportal/handlers/api/instrument.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/handlers/api/instrument.py:971]: I think so. We want people with that level to do this.

**Pull request merged**